docs-v2/content/en/docs/renderers/helm.md

```diff
@@ -92,10 +92,10 @@ manifests:
     releases:
     - name: my-release
       setValueTemplates:
-        image.repository: "{{.myFirstImage.IMAGE_REPO}}"
-        image.tag: "{{.myFirstImage.IMAGE_TAG}}"
-        image2.repository: "{{.mySecondImage.IMAGE_REPO}}"
-        image2.tag: "{{.mySecondImage.IMAGE_TAG}}"
+        image.repository: "{{.IMAGE_REPO.myFirstImage}}"
+        image.tag: "{{.IMAGE_TAG.myFirstImage}}"
+        image2.repository: "{{.IMAGE_REPO.mySecondImage}}"
+        image2.tag: "{{.IMAGE_TAG.mySecondImage}}"
```

Comment:
Hello there! Thanks for the PR! According to the table under the [Image Configuration](https://skaffold.dev/docs/renderers/helm/#image-configuration) section, the values should be, e.g, `{{.IMAGE_TAG_myFirstImage}}`, instead of `{{.IMAGE_TAG.myFirstImage}}`, could you do the change here? Thanks! 😄

Reply:
Done!